minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "server_wildcards": [
+      "play.crystopia.net"
```

Comment:
Doesn't look like a real wildcard

Reply:
You can join under play.crystopia.net bur we have nicht t a real whitecard